lib/ansible/modules/monitoring/monit.py

```diff
@@ -79,16 +80,45 @@ def main():
 
     MONIT = module.get_bin_path('monit', True)
 
+    def monit_version():
+        rc, out, err = module.run_command('%s -V' % MONIT, check_rc=True)
+        version_line = out.split('\n')[0]
+        version = re.search("[0-9]+\.[0-9]+", version_line).group().split('.')
+        # Use only major and minor even if there are more these should be enough
+        return int(version[0]), int(version[1])
+
+    def is_version_higher_than_18():
+        return MONIT_MAJOR_VERSION > 3 or MONIT_MAJOR_VERSION == 3 and MONIT_MINOR_VERSION > 18
```

Comment:
@mangas I think this should have been > 5.18 rather than > 3.18. Could I get you to confirm on https://github.com/ansible/ansible/pull/31212 and give it a shipit/:shipit: if you agree? (I think the PR needs two 'shipit's from community members to be merged.